src/core/evaluator.js

```diff
@@ -232,7 +233,10 @@ class PartialEvaluator {
 
     this._regionalImageCache = new RegionalImageCache();
     this._fetchBuiltInCMapBound = this.fetchBuiltInCMap.bind(this);
-    ImageResizer.setMaxArea(this.options.canvasMaxAreaInBytes);
+    ImageResizer.setOptions({
+      isChrome: this.options.isChrome,
+      maxArea: this.options.canvasMaxAreaInBytes,
+    });
```

Comment:
We could reduce the impact of the new option a little bit in the Firefox PDF Viewer.
```suggestion
    if (typeof PDFJSDev !== "undefined" && PDFJSDev.test("MOZCENTRAL")) {
      ImageResizer.setMaxArea(this.options.canvasMaxAreaInBytes);
    } else {
      ImageResizer.setOptions({
        isChrome: this.options.isChrome,
        maxArea: this.options.canvasMaxAreaInBytes,
      });
    }
```